core/services/vrf/listener_v1.go

```diff
@@ -22,13 +24,16 @@ import (
 	"github.com/smartcontractkit/chainlink/core/services/pg"
 	"github.com/smartcontractkit/chainlink/core/services/pipeline"
 	"github.com/smartcontractkit/chainlink/core/utils"
+	bigmath "github.com/smartcontractkit/chainlink/core/utils/big_math"
 )
 
 var (
 	_ log.Listener   = &listenerV1{}
 	_ job.ServiceCtx = &listenerV1{}
 )
 
+var callbacksTimeout time.Duration = 30 * time.Second
```

Comment:
I think this is a bit too long of a timeout - maybe 5 seconds or 10 seconds should be enough?